File: 프로그래머스/0/181911. 부분 문자열 이어 붙여 문자열 만들기/부분 문자열 이어 붙여 문자열 만들기.py
def solution(my_strings, parts):
    # r=''.join(my_strings[i][s:e+1] for i,(s,e) in enumerate(parts))
    r=''
    for i,(s,e) in enumerate(parts):
        r+=my_strings[i][s:e+1]
    return r

File: 프로그래머스/0/181911. 부분 문자열 이어 붙여 문자열 만들기/부분 문자열 이어 붙여 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(vector<string> my_strings, vector<vector<int>> parts) {
    string r="";
    for(int i=0;i<my_strings.size();i++)
        for(int j=parts[i][0];j<parts[i][1]+1;j++)
            r+=my_strings[i][j];
    return r;
}